WORKSPACE

```diff
@@ -65,3 +65,13 @@ bind(
     name = "gson",
     actual = "@gson_maven//jar",
 )
+
+maven_jar(
+    name = "error_prone_annotations_maven",
+    artifact = "com.google.errorprone:error_prone_annotations:2.3.2",
+)
+
+bind(
+    name = "error_prone_annotations",
+    actual = "@error_prone_annotations_maven//jar",
+)
```

Comment:
Should this also be added to https://github.com/protocolbuffers/protobuf/blob/master/protobuf_deps.bzl?

Reply:
The other java dependencies (guava etc) aren't defined there, so this PR doesn't change anything in that respect.